test/tests/meshgenerators/flip_sideset_generator/tests

```diff
@@ -0,0 +1,40 @@
+[Tests]
+    [flux_2D]
+      type = 'CSVDiff'
+      input = 'flux_flip_2D.i'
+      csvdiff = 'flux_flip_2D_out.csv'
+      recover = false
+      issues = '#25528'
+      design = 'FlipSidesetGenerator.md'
+      requirement = 'This system allows the sideset to be flipped for 2D mesh'
+    []
+
+    [flux_3D]
+      type = 'CSVDiff'
+      input = 'flux_flip_3D.i'
+      csvdiff = 'flux_flip_3D_out.csv'
+      recover = false
+      requirement = 'This system allows the sideset to be flipped for 3D mesh'
+      design = 'FlipSidesetGenerator.md'
+      issues = '#25528'
+    []
+    [no_sideset_exception]
+      type = 'RunException'
+      input = 'no_sideset.i'
+      cli_args = '--mesh-only'
+      expect_err = "sideset doesn't exist in mesh"
+      requirement = 'The system shall error if sideset doesn't exist'
```

Comment:
@MohamedElkamash you'll notice that CIVET is failing your tests, because you have a syntax error here - you are using single quotes for the `requirement`, but you also have a single quote in `doesn't`, so it's ending the string. Can you swap the single quotes to double? Then I think your tests will run.